src/docs/product/issues/issue-details/performance-issues/endpoint-regression/index.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: "Endpoint Regression"
+sidebar_order: 41
+description: "Learn more about Endpoint Regression issues and how to diagnose and fix them."
+---
+
+<Include name="feature-available-for-user-group-early-adopter.mdx" />
+
+Endpoint Regression issues are a generic class of problems where the duration of a transaction increases over time and degrades application performance. Sentry proactively monitors common endpoints out of the box and reports any possible regressions, grouping them as Endpoint Regression issue types.
+
+## Detection Criteria
+
+The detector for performance issues periodically checks the 95th percentile transaction duration of the most common endpoints in your project. When a significant increase in the p95 value is detected and has been sustained for some time, a regression issue is created.
+
+## Regression Evidence
+
+To find additional info, go to the Issues page and click on the Endpoint Regression issue you're interested in. In the top section of the "Details" tab, you'll find the following:
+
+- **Endpoint Name:** The name of the transaction that has regressed.
+- **Change in Duration:** The value of absolute and relative change in duration.
+- **Approximate Start Time:** The approximate time when the regression occurred.
+
+## Chart
+
+The below chart shows the p95 transaction duration over a period of up to 14 days before and 14 days after the regression was identified.
+
+![Endpoint Regression Issue Chart](endpoint-regression-chart.png)
+
+## Potential Causes
+
+The "Potential Causes" section shows a list of spans that may have contributed to the slowdown the most. Next to each span, you'll find the following:
+
+- Span description that leads to the [Span Summary](/product/performance/transaction-summary/) page where you can find more information about that specific span.
+- Throughput of span before and after the observed change in duration.
+- P95 of span self time before and after the observed change in duration.
```

Comment:
This table has changed so we likely want to update it to reflect the new content. You can checkout the function regression issue docs for a reference.

Reply:
done 👍